Gemfile

```diff
@@ -1,12 +1,14 @@
-ruby '3.0.0'
+ruby file: '.ruby-version'
 source 'https://rubygems.org'
 
-gem 'rubocop', '~> 1.8.1'
-gem 'rubocop-rails', '~> 2.9.1'
-gem 'rubocop-performance', '~> 1.9.2'
-gem 'rubocop-rspec', '~> 2.1.0'
+gem 'rubocop', '~> 1.69.2'
+gem 'rubocop-rails', '~> 2.27.0'
+gem 'rubocop-performance', '~> 1.23.0'
+gem 'rubocop-rspec', '~> 3.3.0'
 gem 'rubocop-i18n', '~> 3.0.0'
-gem 'rubocop-rake', '~> 0.5.1' 
-gem 'pronto', '~> 0.10.0' 
-gem 'pronto-rubocop', github: 'prontolabs/pronto-rubocop', ref: '2c16f3f' #, '~> 0.10.0'
+gem 'rubocop-rake', '~> 0.6.0'
+gem 'rubocop-capybara', '~> 2.21.0'
+gem 'pronto', '~> 0.11.2'
+gem 'pronto-rubocop', '~> 0.11.5'
 
+gem 'faraday-retry', '~> 2.2.1'
```

Comment:
For what we should include faraday here?

Reply:
`octokit` dependency. Needed to avoid warning.

Reply:
@dlozano Got it, thanks for explanation!